README.md

```diff
@@ -1,58 +1,169 @@
 # stacks-java [![Build Status](https://dev.azure.com/amido-dev/Amido-Stacks/_apis/build/status/amido-stacks-java-springboot-aks?branchName=master)](https://dev.azure.com/amido-dev/Amido-Stacks/_build/latest?definitionId=101&branchName=master)
 Java Web API Application Scaffolding for Amido Stacks
 
-## Code quality
+## Overview
+This is a sample Java application showcasing
+best coding practices and Integrating with Azure and scaffolding for Amido Stacks
+This application uses [Application Insights](https://docs.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview) for performance management
+[Cosmos DB](https://docs.microsoft.com/en-us/azure/cosmos-db/introduction) is the cloud db used. 
 
-### Formatter
 
-#### Installation
+# Table of Contents
+
+ - [Versioning](#versioning)
+ - [Overview](#overview)
+ - [Table of Contents](#table-of-contents)
+ - Getting Started
+   - [Prerequisites](#prerequisites)
+   - [Building](#building)
+   - [Code quality](#code-quality)
+        - Reports
+            - [Test reports](#test-reports)
+            - [Dependency-Checker](#dependency-check)
+   - [Testing](#testing)
+   - [Docker](#docker)
+ - [IDE Plugins used](#ide-plugins-used)
+ - [Swagger/OAS](#swagger-oas) 
+ - [Health Check](#health-check)
+
+# Licence information
+
+## Versioning
+
+This is the v1.0 of the stacks-java
+
+### Getting Started
+
+#### CONFIGURATION
+
+The following environment variables are required:
+
+- AZURE_COSMOSDB_KEY
+- AZURE_APPLICATION_INSIGHTS_INSTRUMENTATION_KEY
+
+Speak to a member of the team for keys
```

Comment:
For CosmosDB I would hardcode the key in the `application.yml` settings to be the CosmosDB Emulator key and localhost URL, and if people are on Windows it'll just work.

For AppInsights I'd just change the app to not crash if it can't get to AI, and just log to terminal instead locally.

Reply:
(CosmosDB Emulator has a known fixed key)